.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+### Description
+[Describe what this change achieves]
+ 
+### Issues Resolved
+[List any issues this PR will resolve]
+ 
+### Check List
+- [ ] New functionality includes testing.
+  - [ ] All tests pass
```

Comment:
Shouldn't this enforced by the PR check, can we remove this line if its already checked?